schema/field.go

```diff
@@ -36,7 +36,7 @@ func (f *Field) Name() string {
 }
 
 func (f *Field) SimpleName() string {
-	return generator.ToPublicSimpleName(f.avroName)
+	return generator.ToPublicName(f.avroName)
```

Comment:
It's not clear if `Fixed` and `Field` are special in some way that necessitates their need to avoid using the configured namers generation so I swapped them to use `ToPublicName` here. If this isn't correct could you enlighten me as to why these names are different?